tests/functional/RemoteWebElementTest.php

```diff
@@ -117,6 +118,65 @@ public function testShouldClick()
         );
     }
 
+    /**
+     * Test that workarounds to geckodriver#653 works as expected.
+     *
+     * @covers ::click
+     * @covers ::clickChildElement
+     * @dataProvider provideValidClick
+     * @param string $id
```

Comment:
Also makes it sense to limit the tests just to firefox?

Reply:
I'll add `@group exclude-chrome` to the test.